packages/codemirror/src/editor.ts

```diff
@@ -615,7 +631,17 @@ class CodeMirrorEditor implements CodeEditor.IEditor {
       return;
     }
     this._changeGuard = true;
-    this._model.value.text = this.doc.getValue();
+    let value = this._model.value;
+    let start = doc.indexFromPos(change.from);
```

Comment:
Wouldn't we want that? This is performing the change on the pre-changed `IObservableString`, after all.